### PR TITLE
Emit json from rawr processing logging

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/tile-tasks/issues/290

### CODE REVIEW

File: tilequeue/stats.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __call__(self, intersect_metrics, n_enqueued, n_inflight, timing):
                        intersect_metrics['misses'])
 
             pipe.gauge('rawr.process.enqueued', n_enqueued)
-            pipe.gauge('rawr.process.in_flight', n_enqueued)
+            pipe.gauge('rawr.process.inflight', n_inflight)
```
Comment (zerebubuth) [inline tilequeue/stats.py]: Why the change from `rawr.process.in_flight` to `rawr.process.inflight`?

Comment (rmarianski) [reply tilequeue/stats.py]: I just had it written as `inflight` everywhere else in the code so was just trying to be more consistent with that. In this case we won't be searching for it, but I tend to write/guess `inflight` rather than `in_flight` first in searches. But not a stronger reason other than that.
